src/Java/gtPlusPlus/xmod/gregtech/api/metatileentity/implementations/base/GregtechMeta_MultiBlockBase.java

```diff
@@ -1727,6 +1650,9 @@ public void fixAllMaintenanceIssue() {
 		this.mScrewdriver = true;		
 	}
 
+	public boolean checkHatch() {
+		return mMaintenanceHatches.size() <= 1 && !mMufflerHatches.isEmpty();
```

Comment:
Just curious. Is there any non polluting GT++ multi?

Reply:
pss, chemical plant, farmer